public/app/features/alerting/unified/components/rule-editor/DashboardPicker.tsx

```diff
@@ -73,11 +74,12 @@ export const DashboardPicker = ({ dashboardUid, panelId, isOpen, onChange, onDis
 
   const filteredPanels =
     dashboardResult?.dashboard?.panels
-      ?.filter((panel): panel is PanelDTO => typeof panel.id === 'number' && typeof panel.type === 'string')
```

Comment:
I removed the filter here, omitting the panels from the selection seemed confusing for the user